packages/cashier/src/Components/Form/crypto-withdraw-form.jsx

```diff
@@ -123,16 +123,16 @@ const CryptoWithdrawForm = ({
                                 />
                             )}
                         </Field>
-                        <div className='crypto-withdraw-form__percentage-selector'>
-                            <PercentageSelector
-                                amount={+balance}
-                                currency={currency}
-                                getCalculatedAmount={setWithdrawPercentageSelectorResult}
-                                percentage={percentage}
-                                should_percentage_reset={should_percentage_reset}
-                            />
-                        </div>
-                        <div className='crypto-withdraw-form__crypto-fiat-converter'>
+                        <div>
+                            <div className='crypto-withdraw-form__percentage-selector'>
+                                <PercentageSelector
+                                    amount={+balance}
+                                    currency={currency}
+                                    getCalculatedAmount={setWithdrawPercentageSelectorResult}
+                                    percentage={percentage}
+                                    should_percentage_reset={should_percentage_reset}
+                                />
+                            </div>
```

Comment:
@mohammad-rework Could you also remove `__crypto-fiat-converter` from the style file.